e2e/types/manifest.go

```diff
@@ -132,6 +132,12 @@ type Manifest struct {
 
 	// FeatureFlags defines the feature flags to enable.
 	FeatureFlags feature.Flags `toml:"feature_flags"`
+
+	// XCallerEnabled defines whether to enable the xcaller load generator for the monitor app.
+	XCallerEnabled bool `toml:"xcaller_enabled"`
```

Comment:
lets follow the existing loadgen pattern, "Validator self-delegation" is simply enabled by presence of the key. Also, avoid complex configuration options in v1, simply hardcode the chain pairs in `monitor`. lessismore.